gen2tutorialdeprecation.txt

```diff
@@ -0,0 +1,3 @@
+.. warning::
+
+   These tutorials are based on the deprecated Generation 2 CommandlineTask and Butler.
```

Comment:
In my slack message, I had merged the class spelling:  `CmdLineTask` with the spoken english spelling "command-line task."  Should be one or the other.

Reply:
I thought it looked off. I'll fix this and go ahead. 🎉